app/views/rosters/new.haml

```diff
@@ -1,8 +1,16 @@
 %h1.title New roster
 
 = form_for Roster.new do |f|
-  = f.label :name
-  = f.text_field :name
-  = f.submit 'Save'
-  %br
-  To assign the fallback user, first create the roster, then add some users to it.
+  .container-fluid
+    .row
+      .col-xs-4
+      .col-xs-4
+        .form-group
+          = f.label :name
+          = f.text_field :name, class: 'form-control'
+  .center
+    / TODO add functionality so this message isn't necessary
```

Comment:
Meh, unless we'll make it all one screen: creating a roster, adding users, and assigning a fallback.